cypress/cypress-integration/integration/pending-motions-table-judge-and-chamber-dashboard.cy.ts

```diff
@@ -113,20 +117,35 @@ function generateRandomDate() {
   };
 }
 
-function selectRandomStatusOfMatterEntry(): [string, string] {
-  const statusOfMatter = {
-    AwaitingBriefs: 'Awaiting Briefs',
-    AwaitingConsideration: 'Awaiting Consideration',
-    AwaitingRelease: 'Awaiting Release',
-    Drafting: 'Drafting',
-    ReviewingDraft: 'Reviewing Draft',
-    RevisingDraft: 'Revising Draft',
-    Stayed: 'Stayed',
-    SubmittedToChiefJudge: 'Submitted to Chief Judge',
-    SubmittedToReporter: 'Submitted to Reporter',
-  };
-
-  const statusOfMatterEntries = Object.entries(statusOfMatter);
-  const randomIndex = Math.floor(Math.random() * statusOfMatterEntries.length);
-  return statusOfMatterEntries[randomIndex];
+// function selectRandomStatusOfMatterEntry(): [string, string] {
```

Comment:
❓ : Is this implementation still going to be necessary? Perhaps, we can delete

Reply:
Thnx. Totally forgot to do so